src/polyswarm/scan.py

```diff
@@ -42,7 +42,7 @@ def scan():
               help='Does not wait for the scan window to close, just create it and return right away.')
 @click.option('-s', '--scan-config', type=click.STRING, default=None,
               help='Configuration template to be used in the scan. E.g.: "default", "more-time", "most-time".')
-@click.argument('path', nargs=-1, type=click.Path(exists=True))
+@click.argument('path', nargs=-1, type=click.Path(exists=True), required=True)
```

Comment:
consider removing `nargs` if this accepts only one path and can't consume additional filenames as variadic arguments.

Reply:
It can get multiple paths, each one being a different file. Required in this case means that at least one path must be provided.